### PR TITLE
addon:trivy-operator

### PR DESCRIPTION
**Pull request by oeular**
Signed-off-by: Xueliang Yang <yangxueliang@cmbchina.com>

### Checklist

I have:

- [x] Title of the PR starts with type (e.g. `[Addon]` or `[Trait]`).
- [x] Updated/Added any relevant [documentation] and [examples].
- [ ] Unit/E2E Tests added.

### CODE REVIEW

File: addons/trivy-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# trivy-operator
+
```
Comment (wonderflow) [inline addons/trivy-operator/README.md:2]: let's add some description for this addon, how to use？ Why we need it? What's benefit?

File: addons/trivy-operator/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    Healthy Fetch repository successfully, Create helm release successfully
+    No trait applied
+
+vela port-forward addon-trivy-operator -n vela-system
+Forwarding from 127.0.0.1:9115 -> 9115
+Forwarding from [::1]:9115 -> 9115
+Forward successfully! Opening browser ...
+
+kubectl label namespaces vela-system trivy-scan=true
+kubectl label namespaces vela-system trivy-operator-validation=true
```
Comment (wonderflow) [inline addons/trivy-operator/README.md:60]: this command should be something like  :

```
vela env set vela-system --label trivy-scan=true
```

Then we can tell users that this is a environment for development which can scan vulnerabilities.

File: addons/trivy-operator/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+
+curl -s http://localhost:9115/metrics | grep trivy_vulnerabilities | less
```
Comment (wonderflow) [inline addons/trivy-operator/README.md:63]: A better thing need to be done is to Integrate it with grafana dashboard

**Review by wonderflow — COMMENTED** [review]
ping @Somefive  the addon also need observability here.

File: addons/trivy-operator/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+name: trivy-operator
+version: 2.3.2
+description: Kubernetes Operator based on the open-source container vulnerability scanner Trivy
+icon: https://github.com/devopstales/helm-charts/raw/main/icons/trivy.png
+url: https://github.com/devopstales/trivy-operator
```
Comment (wangyikewxgm) [inline addons/trivy-operator/metadata.yaml:5]: This image URL maybe cannot show correctly. Please use `https://raw.githubusercontent.com/devopstales/helm-charts/main/icons/trivy.png`

File: addons/trivy-operator/README.md (hunk 3 above)
Comment (oeular) [reply addons/trivy-operator/README.md:67]: OK. I have a try.

File: addons/trivy-operator/README.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    Type: helm
+    Healthy Fetch repository successfully, Create helm release successfully
+    No trait applied
+
+vela port-forward addon-trivy-operator -n vela-system
+Forwarding from 127.0.0.1:9115 -> 9115
+Forwarding from [::1]:9115 -> 9115
+Forward successfully! Opening browser ...
+
+kubectl label namespaces vela-system trivy-scan=true
```
Comment (wonderflow) [inline addons/trivy-operator/README.md:63]: we have the capability to label a namespace as env now (v1.5), refer to https://github.com/kubevela/kubevela/issues/4180

Please change this to use that feature.

File: addons/trivy-operator/README.md (hunk 3 above)
Comment (wonderflow) [reply addons/trivy-operator/README.md:67]: we have grafana addon now, you can just re-use that.

File: addons/trivy-operator/README.md (hunk 5 above)
Comment (oeular) [reply addons/trivy-operator/README.md:63]: ok, I will refine it.

File: addons/trivy-operator/README.md (hunk 3 above)
Comment (oeular) [reply addons/trivy-operator/README.md]: Only import the json files to the granafa, so write the json files url ( https://github.com/devopstales/trivy-operator/tree/main/grafana-dashboard) to the README.md ?

**Review by wonderflow — APPROVED** [review]

**Review by wonderflow — COMMENTED** [review]
please move to experimental

**Pull request merged**